src/main/java/uk/ac/cam/cl/dtg/segue/configuration/SegueGuiceConfigurationModule.java

```diff
@@ -460,6 +461,38 @@ private static EmailManager getMessageCommunicationQueue(final IUserDataManager
         return emailCommunicationQueue;
     }
 
+    /**
+     * This provides a singleton of the UserManager for various facades.
+     *
+     * Note: This has to be a a singleton as the User Manager keeps a temporary cache of anonymous users.
+     *
+     * @param database
+     *            - the user persistence manager.
+     * @param properties
+     *            - properties loader
+     * @param providersToRegister
+     *            - list of known providers.
+     * @param emailQueue
+     *            - so that we can send e-mails.
+     * @param mapperFacade
+     *            - for DO and DTO mapping.
+     * @return Content version controller with associated dependencies.
+     */
+    @Inject
+    @Provides
+    @Singleton
```

Comment:
I am not sure what about this change means this now has to be a singleton?

Reply:
It'll be created once per WebSocket request otherwise, ~50k times a week, since it's injected into each WebSocket object and one is made per request. I think? @sacummins